install

```diff
@@ -61,6 +62,7 @@ print_usage_with_options() {
   echo "  --with[out]-jdk              Install JDK instead of using system version (default: use system version)"
   echo "  --with[out]-ant              Install ant instead of using system version (default: use system version)"
   echo "  --with[out]-claw             Install CLAW and OMNI Compiler (default: disabled)"
+  echo "  --with[out]-omni             Install OMNI Compiler; cannot be combined with --with-claw (default: disabled)"
```

Comment:
Maybe prefer that one in the `INSTALL.md` now in the example with `--hpc2020`?

Reply:
Thanks, I have update the instructions and also added the missing ability to install now OMNI instead of CLAW+OMNI via CMake. Please have another look